contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -425,7 +425,10 @@ Conditions are most useful when they follow some consistent conventions:
 Conditions should follow the standard schema included in [k8s.io/apimachinery/pkg/apis/meta/v1/types.go](https://github.com/kubernetes/apimachinery/blob/release-1.23/pkg/apis/meta/v1/types.go#L1432-L1492).
 It should be included as a top level element in status, similar to
 ```go
-Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type" protobuf:"bytes,1,rep,name=conditions"`
+// +listType=map
+// +listMapKey=type
```

Comment:
I would include these as well to keep client-side apply working as we want
```go
	// +patchStrategy=merge
	// +patchMergeKey=type
```

Reply:
Done. I'll do a sweep separately.